app/javascript/Components/Result/file_viewer.jsx

```diff
@@ -262,4 +79,18 @@ export class FileViewer extends React.Component {
       return "";
     }
   }
+
+  render() {
+    const viewer = this.getViewer();
+
+    return (
+      <React.Fragment>
+        <div style={{display: this.state.loading || this.state.errorMessage ? "none" : "block"}}>
```

Comment:
This extra div is causing a rendering problem with the PDF viewer (height collapsing).

Fixing CSS is a bit tricky; instead of wrapping in a div, you can pass a `display` prop to each viewer, and use that to control the display of each component.